test/Microsoft.VisualStudio.LanguageServices.Razor.Test/Serialization/TagHelperDescriptorSerializationTest.cs

```diff
@@ -18,6 +18,51 @@ public void TagHelperDescriptor_RoundTripsProperly()
         {
             // Arrange
             var expectedDescriptor = CreateTagHelperDescriptor(
+                kind: TagHelperConventions.DefaultKind,
```

Comment:
Can we use something other than the default kind in the test? This test would pass even if you never fixed the bug

Reply:
Oh NVM I see you added a bonus test